examples/gallery/image_bright_regions_gallery_example.py

```diff
@@ -0,0 +1,95 @@
+
+# coding: utf-8
+
+"""
+====================================================
+Manipulating Map image data - finding bright regions
+====================================================
+
+This example shows how you can do basic image processing on SunPy map image data. In this example, we try to find the brightest regions in an AIA image, and count the approximate number of regions of interest.
+"""
+
+##############################################################################
+# First, import the modules we will need:
+
+
+import sunpy
+from sunpy import map
```

Comment:
don't do this!!!! rather:
```python
import sunpy.map
```
Otherwise you are killing the `map` function in main python